### PR TITLE
feat(sdk-coin-sui): create broadcastable MPC transaction

### PR DESCRIPTION
**Pull request by bhavidhingra**
TICKET: [WIN-3293](https://bitgoinc.atlassian.net/browse/WIN-3293)

[WIN-3293]: https://bitgoinc.atlassian.net/browse/WIN-3293?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .gitcommitscopes [hunk 1]
```diff
@@ -0,0 +1 @@
+sdk-coin-sui
```
Comment (zahin-mohammad) [inline .gitcommitscopes:1]: curious on what this file does

File: modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts [hunk 2]
```diff
@@ -455,6 +457,8 @@ export interface MPCRecoveryOptions {
   walletPassphrase?: string;
   seed?: string;
   index?: number;
+  startingScanIndex?: number;
+  scan?: number;
```
Comment (zahin-mohammad) [inline modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts:461]: Please add jsdocs for these three fields. How is index different from scan?

File: modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts [hunk 3]
```diff
@@ -445,6 +445,8 @@ export interface MPCTx {
     commonKeychain?: string;
     lastScanIndex?: number;
   };
+  recoveryAmount?: string;
+  signature?: string;
```
Comment (zahin-mohammad) [inline modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts:449]: jsdoc herw would also be useful

File: .gitcommitscopes (hunk 1 above)
Comment (bhavidhingra) [reply .gitcommitscopes]: remove it

File: modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts (hunk 2 above)
Comment (bhavidhingra) [reply modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts]: `index` and `startingScanIndex` both serve the same purpose, some WRW flows used `index` while others use `startingScanIndex`.

File: modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts (hunk 3 above)
Comment (bhavidhingra) [reply modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts]: done

File: modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts [hunk 4]
```diff
@@ -445,6 +445,10 @@ export interface MPCTx {
     commonKeychain?: string;
     lastScanIndex?: number;
   };
+  // the amount recovered using WRW
+  recoveryAmount?: string;
+  // the transaction signature used for broadcasting the transaction
+  signature?: string;
```
Comment (zahin-mohammad) [inline modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts:451]: How is this different from `serializedTx`? Is the idea to deprecate that field? 
If you look at the sol recovery implementation, the `serializedTx` field is used to return the signed tx ready for broadcast

Comment (bhavidhingra) [reply modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts:451]: To broadcast a Sui transaction, we need both `serializedTx` and `signature`.

**Review by zahin-mohammad — APPROVED** [review]

**Review by hitansh-madan — APPROVED** [review]

**Pull request merged**